Exercise-master/src/test/java/org/upgrad/controllers/UserControllerTest.java

```diff
@@ -67,9 +69,9 @@ public void registerUser() throws Exception{
 
     @Test
     public void registerUserWithExistingUsername() throws Exception{
-        User user= new User();
-       user.setId(1);
-       user.setUserName("upgrad");
+        Users user= new Users();
```

Comment:
In the original file I see User user = new User.  Are we sure if we make this change it will upgrad's review?